src/convention.md

```diff
@@ -0,0 +1,7 @@
+# `convention.md` - How should your file be named ?
```

Comment:
this file looks like a duplicate of `src/commit.md`

Reply:
That is not a duplicate. I use two conventions, although very similar. There is one convention for the naming of the file, and another for the commit used to save said files.

Reply:
mm, these are actually the same file, only the header changes, but the list is the exact same one :thinking: 
maybe i misunderstand something here though